docs/guides/vqa_guide/structure.rst

```diff
@@ -47,11 +45,26 @@ We'll go through the steps as they were outlined above:
 
 1. The ``Ansatz``, ``Backend`` and ``Optimizer`` can be defined independently of one another.
 2. As explained in :ref:`estimators guide <estimators>`, the ``EstimationTaskFactory`` tells the quantum computer how to construct a circuit. It therefore will always take an ``Ansatz`` as an argument.
-3. As explained briefly above, VQA uses a quantum computer to estimate the cost function. So it is sensible that to create our cost function we would need the method of costructing circuits for our cost function (``EstimationTask``) and the quantum computer to run them on (``QuantumBackend``). See the :ref:`cost function <cost_function>` guide for details.
+3. As explained briefly above, VQA uses a quantum computer to estimate the value of the cost function. So to create our cost function we would need the method of costructing circuits for our cost function (``estimation_task_factory``) and the quantum computer to run them on (``QuantumBackend``). See the :ref:`cost function <cost_function_guide>` guide for details.
 4. Now that we have the , we minimize if using the optimizer. This step generally takes the longest to run.
```

Comment:
4. is incomplete

Reply:
Fixed in https://github.com/zapatacomputing/orquestra-core/pull/60/commits/0b3d2fd9b5438795d3280b6ecad3885360af1ba2